components/Chat/ChatPlaceholder/ChatPlaceholder.tsx

```diff
@@ -81,8 +82,7 @@ export default function ChatPlaceholder({ messagesCount }: Props) {
                     options: ["Unblock", "Cancel"],
                     cancelButtonIndex: 1,
                     destructiveButtonIndex: isBlockedPeer ? undefined : 0,
-                    title:
-                      "If you unblock this contact, they will be able to send you messages again.",
+                    title: translate("if_you_unblock_contact"),
```

Comment:
😃

Reply:
@alexrisch thanks for the review 🙏 we've got to translate those strings 😁